sphinx/tutorial/cylc/furthertopics/inheritance.rst

```diff
@@ -0,0 +1,603 @@
+   HELICOPTER   [color=royalblue,
+      fillcolor=powderblue,
+      shape=box,
+      style=filled];
+   HELICOPTER -> v22  [color=royalblue];
+   r44    [color=royalblue,
+      fillcolor=powderblue,
+      shape=box,
+      style=filled];
+   HELICOPTER -> r44  [color=royalblue];
+   root   [color=royalblue,
+      fillcolor=powderblue,
+      shape=box,
+      style=filled];
+   VEHICLE   [color=royalblue,
+      fillcolor=powderblue,
+      shape=box,
+      style=filled];
+   root -> VEHICLE    [color=royalblue];
+   AIR_VEHICLE  [color=royalblue,
+      fillcolor=powderblue,
+      shape=box,
+      style=filled];
+   VEHICLE -> AIR_VEHICLE   [color=royalblue];
+   AIR_VEHICLE -> AIRPLANE  [color=royalblue];
+   AIR_VEHICLE -> HELICOPTER   [color=royalblue];
+
+Inspect the configuration of the ``v22`` task using the ``cylc get-config``
+command.
+
+.. spoiler:: Hint warning
+
+   .. code-block:: bash
+
+      cylc get-config . --sparse -i "[runtime][v22]"
+
+You should see that the ``CAN_TASK_OFF_VERTICALLY`` environment variable has
+been set to ``false`` which isn't right. This is because of the order in which
+inheritance is applied.
+
+Cylc handles multiple-inheritance by applying each family from last to first.
+For the ``v22`` task we specified ``inherit = AIRPLANE, HELICOPTER`` so the
+``HELICOPTER`` family will be applied first and the ``AIRPLANE`` family after.
+
+The inheritance order would be as follows:
+
+.. code-block:: bash
+
+   root
+   VEHICLE
+   AIR_VEHICLE
+   HELICOPTER   # sets "CAN_TAKE_OFF_VERTICALLY to "true"
+   AIRPLANE     # sets "CAN_TAKE_OFF_VERTICALLY to "false"
+   v22
+
+We could fix this problem by changing the order of inheritance:
+
+.. code-block:: cylc
+
+   inherit = HELICOPTER, AIRPLANE
+
+Now the ``HELICOPTER`` family is applied second so its values will override any
+in the ``AIRPLANE`` family.
+
+.. code-block:: bash
+
+   root
+   VEHICLE
+   AIR_VEHICLE
+   AIRPLANE     # sets "CAN_TAKE_OFF_VERTICALLY to "false"
+   HELICOPTER   # sets "CAN_TAKE_OFF_VERTICALLY to "true"
+   v22
+
+Inspect the configuration of the ``v22`` task using ``cylc get-config`` to
+confirm this.
+
+
+More Inheritance
+----------------
+
+We will now add some more families and tasks to the suite.
+
+Engine Type
+^^^^^^^^^^^
+
+Next we will define three families to represent different types of engine.
+
+.. digraph:: Example
+   :align: center
+
+   size = "5,5"
+   bgcolor=none
+
+   ENGINE [color=royalblue, fillcolor=powderblue, shape=box, style=filled]
+   TURBINE_ENGINE [color=royalblue, fillcolor=powderblue, shape=box, style=filled]
+   INTERNAL_COMBUSTION_ENGINE [color=royalblue, fillcolor=powderblue, shape=box, style=filled]
+   HUMAN_ENGINE [color=royalblue, fillcolor=powderblue, shape=box, style=filled]
+
+   ENGINE -> TURBINE_ENGINE
+   ENGINE -> INTERNAL_COMBUSTION_ENGINE
+   ENGINE -> HUMAN_ENGINE
+
+Each engine type should set an environment variable called ``FUEL`` set to the
```

Comment:
Change the former instance of 'set' to 'be assigned' or similar, as the first clause doesn't particularly make sense and I assume is a typo/error.